src/instructlab/sdg/pipeline.py

```diff
@@ -193,10 +194,11 @@ def _generate_single(self, dataset) -> Dataset:
                 block_config = block_prop["config"]
                 drop_columns = block_prop.get("drop_columns", [])
                 drop_duplicates_cols = block_prop.get("drop_duplicates", False)
+                if block_type == llmblock.LLMBlock:
+                    block_config["gen_kwargs"]["max_tokens"] = self.ctx.max_num_tokens
```

Comment:
Can you move this to happen within LLMBlock's __init__ instead?

Reply:
added to `LLMBlock` so that if not None, we use the value provided, else use the value from the pipeline yaml